charts/dapr/charts/dapr_sidecar_injector/templates/dapr_sidecar_injector_deployment.yaml

```diff
@@ -154,6 +154,11 @@ spec:
 {{- if not .Values.global.actors.enabled }}
         - name: SKIP_PLACEMENT
           value: "true"
+{{- else }}
+{{- if .Values.global.ha.enabled }}
```

Comment:
This needs to take into account the cluster DNS suffix, it may not always be `cluster.local`. Must use ``.Values.global.dnsSuffix``

However you may be able to get away without hardcoding this, by doing a DNS resolution on the service, assuming it has ClusterIP=None?

Reply:
OK. I will try the dnsSuffix.

Reply:
Using dnsSuffix now.

Reply:
Removed since Josh showed me that this is handled in the sidecar via dns resolution.